packages/distribution/translations/it.yaml

```diff
@@ -68,7 +68,7 @@ caluma:
       search: "Cerca..."
       suggestions: "Raccomandazioni"
       empty: "Non sono stati trovati servizi"
-      groups: "{count} {count, plural, =1 {servizio} other {servizi}}"
+      groups: "{count} {count, plural,= 1 {servizio} other {servizi}}"
```

Comment:
Did you try to start the application? I vaguely remember having had to fix this one during the last import.

Reply:
You were right. Now that I have changed it, the tests run through. :+1: